spec/support/email.rb

```diff
@@ -1,4 +1,8 @@
 RSpec.configure do |config|
   config.include(EmailSpec::Helpers)
   config.include(EmailSpec::Matchers)
+
+  config.before(:each) do
+    ActionMailer::Base.deliveries.clear
+  end
```

Comment:
Maybe we should include [`EmailSpec::MailerDeliveries`][1] and use that instead of dealing with `ActionMailer::Base.deliveries`, e.g:

```
config.before(:each) do
  reset_mailer
end
```

and

```
expect { perform_job }.to change(all_emails, :size).by(1)
```



[1]: https://github.com/bmabey/email-spec/blob/master/lib/email_spec/deliveries.rb